bin/utils/index.js

```diff
@@ -15,7 +16,24 @@ async function npmInstall (cwd, dependencies) {
     })
 }
 
+function splitSemvarVersion (version) {
+  const versionParts = version.split('.').map(Number)
```

Comment:
does this still cope with '^13'?

Reply:
I'm being more accurate by reading it directly from the installed package rather than the package.json.  `^13.0.0` could point to `13.0.0`, `13.0.1`, `13.1.0`, and soon `13.2.0` but what we care about is the actually installed version so we're now reading that from `node_modules/<package-name>/package.json` which is only going to be a precise version number e.g. `13.0.1`.

Reply:
ah cool